lib/site_template/404.html

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+---
+
+<style type="text/css" media="screen">
+  .container {
+    margin: 10px auto;
+    width: 600px;
+    text-align: center;
+  }
+  h1 {
+    width: 800px;
+    position: relative;
+    left: -100px;
+    margin: 30px 0;
+    font-size: 60px;
```

Comment:
One commonly used blog template should be mobile-first and responsive, even here.